### PR TITLE
feat!: get ready to release

### PR DESCRIPTION
**Pull request by marrip**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+drmaa==0.7.9
+hydra-genetics==v0.8.0
```
Comment (jonca79) [inline requirements.txt:2]: There is a 0.9.0 now. Should it be v before?

File: workflow/schemas/config.schema.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    description: path to samples.tsv file
+  units:
+    type: string
+    description: path to units.tsv file
+  default_container:
+    type: string
+    description: name or path to a default docker/singularity container
+  spring:
+    type: object
+    description: parameters for fastp_pe
```
Comment (jonca79) [inline workflow/schemas/config.schema.yaml:19]: Wrong description

File: workflow/schemas/resources.schema.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        description: max execution time by default
+    required:
+      - mem_mb
+      - mem_per_cpu
+      - partition
+      - threads
+      - time
+  spring:
+    type: object
+    description: resource definitions for fastp_pe
```
Comment (jonca79) [inline workflow/schemas/resources.schema.yaml:32]: wrong description

File: requirements.txt (hunk 1 above)
Comment (marrip) [reply requirements.txt:2]: you are completely right

Comment (marrip) [reply requirements.txt:2]: v is not required for some reason 🤔

**Review by Smeds — APPROVED** [review]

File: workflow/schemas/config.schema.yaml (hunk 2 above)
Comment (marrip) [reply workflow/schemas/config.schema.yaml:19]: changed it

File: workflow/schemas/resources.schema.yaml (hunk 3 above)
Comment (marrip) [reply workflow/schemas/resources.schema.yaml:32]: changed it

**Review by jonca79 — APPROVED** [review]

**Pull request merged**